src/mumble/ConfigWidget.cpp

```diff
@@ -60,6 +61,16 @@ void ConfigWidget::loadCheckBox(QAbstractButton *c, bool v) {
 	}
 }
 
+void ConfigWidget::loadRadioButton(QAbstractButton *c, bool v) {
+	if (v != c->isChecked()) {
+		c->setChecked(v);
+	} else {
+		connect(this, SIGNAL(boolSignal(bool)), c, SIGNAL(toggled(bool)));
+		emit boolSignal(v);
+		disconnect(SIGNAL(boolSignal(bool)));
```

Comment:
What are you trying to do here? This seems like you could just call `toggled` directly without going through the signal/slot-fuzz :eyes:

Reply:
This was not clear to me too. But I just copied the same pattern from loadCheckBox(). But anyways I have changed it to your suggestion.

Reply:
Done. Replaced all these with toggled(value);